plugins/init/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@reframe/init",
+    "version": "0.0.1-rc.12",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": "github:reframejs/reframe"
+}
```

Comment:
We should move the deps as well.  E.g.`inquirer` is now loaded by `@reframe/init`